### PR TITLE
add t3 instance type settings

### PR DESCRIPTION
**Pull request by rochacon**
## Description

AWS announced [1] new t3 instance types, this PR adds the necessary configuration to use them. 

[1] https://aws.amazon.com/blogs/aws/new-t3-instances-burstable-cost-effective-performance/

### Checklist

- [x] `terraform fmt` and `terraform validate` both work from the root and `examples/eks_test_fixture` directories (look in CI for an example)
- [ ] Tests for the changes have been added and passing (for bug fixes/features)
- [ ] Test results are pasted in this PR (in lieu of CI)
- [ ] Docs have been updated using `terraform-docs` per `README.md` instructions
- [x] I've added my change to CHANGELOG.md
- [x] Any breaking changes are highlighted above

### CODE REVIEW

File: local.tf [hunk 1]
```diff
@@ -178,6 +183,13 @@ locals {
     "t2.nano"      = false
     "t2.small"     = false
     "t2.xlarge"    = false
+    "t3.2xlarge"   = false
+    "t3.large"     = false
+    "t3.medium"    = false
+    "t3.micro"     = false
+    "t3.nano"      = false
```
Comment (max-rocket-internet) [inline local.tf]: Maybe don't include nano and micro here if they aren't in the `max_pod_per_node` map also?

Comment (rochacon) [reply local.tf]: Agreed. I basically duplicated the t2 block and replaced t2 with t3.

Should I drop the t2.nano and t2.micro from this list also?

Comment (max-rocket-internet) [reply local.tf]: Ah I see what you did. Yes, I think make your change consistent. Don't worry about the other values.

Comment (rochacon) [reply local.tf]: Got it. It's done.

Comment (strygin) [reply local.tf]: @max-rocket-internet 
If you request changes in your "reviews", please try to inspect the code more deeply. This is not a linux kernel repo and your analysis is too superficial. What you do here is actually slowing the progress down.

**Comment by rochacon** [comment]
Thank you @max-rocket-internet!! Any idea on the release date to 1.5.0? I'm also interested in the monitoring flag and spot instances.

File: local.tf (hunk 1 above)
Comment (brandoconnor) [reply local.tf]: Good catch on these @rochacon and I agree with what you've suggested @max-rocket-internet . I've been experimenting with a way we can start relying on [this utility module](https://github.com/terraform-aws-modules/terraform-aws-ebs-optimized) for the `ebs_optimized` portion and will pitch sourcing it from there.

@strygin - Max's comments here are the furthest thing from pedantic or taste-based. Again, see my feedback elsewhere and prioritize being respectful, thoughtful, and on discussion in future comments.

**Comment by brandoconnor** [comment]
Apologies I haven't released sooner @rochacon . I might have time this weekend but I've been super busy as of late. @max-rocket-internet if you've got the bandwidth, feel free to get a release PR up so we can discuss and ship it.

**Comment by max-rocket-internet** [comment]
> Thank you @max-rocket-internet!! Any idea on the release date to 1.5.0?

I'll do it this week. Tomorrow or Wednesday 🙂

**Pull request merged**